proto/v1/swagger/image.swagger.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Let's remove all the swagger specs for now, as we might not be generating client libraries from Swagger spec going forward.